Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM yandex/clickhouse-client:19.5.3.8
+
+RUN apt-get update && apt-get install -y curl python python-pip && pip install elasticsearch-loader==0.2.25
+COPY . ./genetics-backend
+COPY ./clickhouse_client_settings.xml /etc/clickhouse-client/conf.d/
+
+ENTRYPOINT [ "/bin/bash" ]
```

Comment:
hi @forus , maybe it is also good to add something to the README (or perhaps in comment lines here) about the purpose of this Dockerfile and how to use it.

Reply:
Done

Reply:
@forus thanks. What about instructions on how to use it? Think of `docker build...`, `docker run...` commands and also what one should change in https://github.com/opentargets/genetics-backend/blob/master/loaders/clickhouse/create_and_load_everything_from_scratch.sh to use this container.

Reply:
I've just documented in the readme the steps. You don't have to modify the script to be able to run it in the container (all systems assumed to run under `localhost` in this version). https://github.com/opentargets/genetics-backend/pull/11 adds a possibility to specify the hosts with env. variables.